test/test-golangci-lint.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# print current running test
+echo running "$0"
+
+command -v golangci-lint >/dev/null 2>&1 || { echo >&2 "golangci-lint not found"; exit 1; }
+
+# configure settings for test scripts
+ROOT=$(dirname "${BASH_SOURCE}")/..
+cd "${ROOT}" # Enter mgmt root
```

Comment:
nit: comments are either full sentences and are capitalized and end with a period or aren't capitalized and don't end with a period.